app/components/Login/Login.jsx

```diff
@@ -70,12 +70,11 @@ export default class Login extends React.Component {
                 "Creds": {
                     "Type": this.state.loginMethodType,
                     "Username": this.state.username,
-                    "password": this.state.password
+                    "Password": this.state.password
                 }
             })
                 .then((resp) => {
```

Comment:
We need to add an LDAP login methodtype to the login API here: https://github.com/djenriquez/vault-ui/blob/master/src/login.js#L29 because LDAP uses a different endpoint for authentication. Without this change the API will return a 400: "Invalid auth method".